app/views/report/_form_filter_chargeback.html.haml

```diff
@@ -152,12 +152,13 @@
     %label.control-label.col-md-2
       = _('Group by')
     .col-md-8
-      - opts = [["#{_('Date')}", "date"], ["#{_(@edit[:new][:cb_model])}", "vm"]]
-      - opts += [["#{_('Tag')}", "tag"]] unless @edit[:new][:model] == "ChargebackContainerImage" || @edit[:new][:model] == "MeteringContainerImage"
+      - opts = [[_("Date and #{@edit[:new][:cb_model]}"), "date"]]
```

Comment:
`#{}` inside `_()` won't work. You need to use `_("... %{chargeback_model}") % {:chargeback_model => @edit...}`